### DISCUSSION TITLE
Checksum error when importing from CLI

### DISCUSSION THREAD

**Discussion by hindmarshcc** [photoprism/photoprism#4133] [Q&A]
Importing via CLI suddenly stopped working for me and I am trying to determine why and what changed.  Below is the output of the import command.  For reference, I am running Photoprism as a plugin on an Openmediavault instalation.  Importing via the GUI works as expected.

time="2024-03-30T16:24:30Z" level=info msg="Become a member today, support our mission and enjoy our member benefits! 💎"
time="2024-03-30T16:24:30Z" level=info msg="Visit https://www.photoprism.app/membership to learn more."
time="2024-03-30T16:24:31Z" level=info msg="moving media files from /photoprism/import to /photoprism/originals"
time="2024-03-30T16:24:31Z" level=info msg="classify: loading nasnet"
time="2024-03-30T16:24:34Z" level=error msg="import: Checksum does not match: stored 3707234832 vs. calculated on the restored bytes 2221178510\n\t [[{{node save/RestoreV2}}]]"
time="2024-03-30T16:24:34Z" level=info msg="completed in 4.009568931s"

Thank You in advance.

**Comment by lastzero** [comment]
This error seems to come from TensorFlow, probably because the [NASNet model](https://dl.photoprism.app/tensorflow/) in the *assets* folder could not be loaded. It is part of our distribution (Docker image) and the only cause I can think of is that it was manually modified or that there is a hardware/software issue with your disks.
